modules/rhcos-install-iscsi-ibft.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_bare_metal/installing-bare-metal.adoc
+// * installing/installing_bare_metal/installing-bare-metal-network-customizations.adoc
+// * installing/installing_bare_metal/installing-restricted-networks-bare-metal.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="rhcos-install-iscsi-ibft_{context}"]
+= Installing {op-system} on an iSCSI boot device using iBFT
+
+On a completely diskless machine, the iSCSI target and initiator values can be passed through iBFT. iSCSI multipathing is also supported.
+
+.Prerequisites
+. You are in the {op-system} live environment.
+. You have an iSCSI target you want to install {op-system} on.
+. Optional: you have multipathed your iSCSI target.
+
+.Procedure
+
+. Mount the iSCSI target from the live environment:
++
+[source,text]
+----
+$ iscsiadm \
+    --mode discovery \
+    --type sendtargets
+    --portal 192.168.0.10 \ <1>
+    --login
+----
+<1> The IP address of the target portal.
+
+. Optional: enable multipathing and start the daemon:
++
+[source,text]
+----
+$ mpathconf --enable && systemctl start multipathd.service
+----
+
+. Install {op-system} onto the iSCSI target using the necessary kernel arguments, specifying `rd.iscsi.firmware=1`:
++
+[source,text]
+----
+$ coreos-installer install \
```

Comment:
Multiline commands don't copy properly on the portal; if you know of another existing section already published where you have commands like this, you might want to check them on docs.redhat.com and see what the copy/paste behavior is. Leaving off "$" fixes the error on the CP. I haven't verified on the docs.rh.com.

Reply:
that's really unfortunate :/ it forces us to make the choice to sacrifice readability and UX for the sake of working around a UI technical limitation. I'm inclined to leave it as is because i think it's more important than being able to copy to the clipboard properly esp when the option to manually type it in exists